### PR TITLE
Improve colors of horizontal scrollbar

### LINKED ISSUE #40: Helmholtz Scrollbar not shown on iOS Safari and Linux Firefox
The Helmholtz Scrollbar in green is not visible on:

* iOS Safari (iOS 15.5)
* Firefox (Linux, v103.0)

### PR DESCRIPTION
**Pull request by ruester**
# Fix colors of horizontal scrollbar

Fixes #40

Changes proposed in this pull request:

* change the color of the scrollbar for Firefox
* change the hover color of the scroll buttons

Now the participating organisations scroll component looks the same in Firefox and Chrome:

image

How to test:

* build this branch and use the scroll component on the front page

### CODE REVIEW

**Review by cmeessen — CHANGES_REQUESTED** [review]
So, the good thing is: it works on MacOS Safari.

The bad thing: it doesn't work on iOS Safari.

And then, there are some points regarding Firefox:

* the scroll bar is hidden by default
* the scroll bar is too thin
* the scroll bar jumps around when hovering
* the scroll bar edges are rounded and not at right angle
* there is a white space between the scrollbar and the lower border of the div

Sorry :/ Do you think you can find a way to fix this?

**Comment by ruester** [comment]
I finally got it working by using the library [simplebar-react](https://www.npmjs.com/package/simplebar-react). I've tested this new solution also on my mobile with chrome and firefox and it works. @cmeessen please test it again for iOS/Safari

**Comment by ruester** [comment]
Chrome:
image

Firefox:
image

The scrollbar from the Helmholtz homepage has `border-radius: 10px` which I also used for our scrollbar now. So it has *not* to be right angle.

From helmholtz.de:
image

File: frontend/pages/index.tsx [hunk 1]
```diff
@@ -230,58 +233,97 @@ function ResearchFields() {
...
-          )
-        })
-      }
-      </div>
+      <SimpleBar
+        id="hgf-simplebar"
+        ref={sbRef}
+        autoHide={false}
+        forceVisible="x"
+        style={{maxHeight: 300}}
```
Comment (cmeessen) [inline frontend/pages/index.tsx:293]: `#hgf-scroll-container` is already set to 200px. Should we set this here too?

File: frontend/styles/custom.css [hunk 2]
```diff
@@ -33,19 +33,31 @@ header input {
...
-.hgf-scrollbar {
+
+#hgf-scroll-container {
+  display: flex;
+  height: 200px;
+}
+
+.simplebar-track.simplebar-horizontal {
+  height: 7px !important;
   border-bottom: 1px solid #002864;
```
Comment (cmeessen) [inline frontend/styles/custom.css:48]: The bottom border was missing and I added it again. However, there is a 2 pixel offset at both ends. Is there any way to fix this?

image

File: frontend/styles/custom.css [hunk 3]
```diff
@@ -33,19 +33,31 @@ header input {
...
-.hgf-scrollbar::-webkit-scrollbar {
-  height: 5px;
+
+.simplebar-horizontal .simplebar-scrollbar {
+  height: 5px !important;
+}
+
+.simplebar-horizontal .simplebar-scrollbar::before {
+  background-color: #05e5ba !important;
+  border-radius: 0px !important;
```
Comment (cmeessen) [inline frontend/styles/custom.css:57]: > The scrollbar from the Helmholtz homepage has border-radius: 10px which I also used for our scrollbar now. So it has not to be right angle.

The official Helmholtz brand style guide proposes a right angle scroll bar, so I changed it back.

**Review by cmeessen — CHANGES_REQUESTED** [review]
Good job, we are almost there. It works on iOS Safari as well as on Firefox :+1:

There are a few more things to be done (see comments). Also, the width of the scrollbar is rather small on mobile:

image

Having it about 1/3 of the container width would be perfect.

**Pull request merged**